lib/composer.json

```diff
@@ -4,7 +4,8 @@
     "minimum-stability": "dev",
     "require-dev": {
         "league/container": "4.2",
-        "coenjacobs/mozart": "0.7.1"
+        "coenjacobs/mozart": "0.7.1",
+        "psr/container": "^1.1"
```

Comment:
I would replace caret and use a fixed version instead, like `"psr/container": "1.1.2"`

Reply:
+1 to this

Reply:
+1 to this